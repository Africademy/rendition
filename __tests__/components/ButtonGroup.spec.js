/* globals expect, test */
import React from 'react'
import renderer from 'react-test-renderer'
import {
  Button,
  ButtonGroup,
  Provider
} from '../../dist'

test('ButtonGroup renders correctly', () => {
  const component = renderer.create(
    <Provider>
      <ButtonGroup>
        <Button>First</Button>
        <Button>Second</Button>
        <Button>Third</Button>
      </ButtonGroup>
    </Provider>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
