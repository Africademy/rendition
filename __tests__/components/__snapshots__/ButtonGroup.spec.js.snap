// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonGroup renders correctly 1`] = `
<div
  className="sc-btzYZH jBDBBH StyledGrommet-sc-19lkkz7-0 bBubtL"
>
  <div
    className="sc-htoDjs sc-cMljjf dQmdry sc-gzVnrw pznRG sc-bZQynM kMvugX"
  >
    <button
      className="sc-kEYyzF sc-brqgnP fjrEUv sc-iAyFgw kTGuvf StyledButton-sc-323bzc-0 djTlJn"
      onBlur={[Function]}
      onFocus={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      type="button"
    >
      First
    </button>
    <button
      className="sc-kEYyzF sc-brqgnP fjrEUv sc-iAyFgw kTGuvf StyledButton-sc-323bzc-0 djTlJn"
      onBlur={[Function]}
      onFocus={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      type="button"
    >
      Second
    </button>
    <button
      className="sc-kEYyzF sc-brqgnP fjrEUv sc-iAyFgw kTGuvf StyledButton-sc-323bzc-0 djTlJn"
      onBlur={[Function]}
      onFocus={[Function]}
      onMouseOut={[Function]}
      onMouseOver={[Function]}
      type="button"
    >
      Third
    </button>
  </div>
</div>
`;
